runtime/fundamentals/configuration.md

```diff
@@ -0,0 +1,419 @@
+---
+title: "deno.json and package.json"
+oldUrl: /runtime/manual/getting_started/configuration_file/
+---
+
+You can configure Deno using a `deno.json` file. This file can be used to
+configure the TypeScript compiler, linter, formatter, and other Deno tools.
+
+The configuration file supports `.json` and
+[`.jsonc`](https://code.visualstudio.com/docs/languages/json#_json-with-comments)
+extensions.
+
+Deno will automatically detect a `deno.json` or `deno.jsonc` configuration file
+if it's in your current working directory or parent directories. The `--config`
+flag can be used to specify a different configuration file.
+
+An example `deno.json` file:
+
+:::info Version notes
+
+- Before Deno v1.23, you needed to supply an explicit `--config` flag.
+- Starting with Deno v1.34, globs are supported in `include` and `exclude`
+  fields. You can use `*` to match any number of characters, `?` to match a
+  single character, and `**` to match any number of directories.
+
+:::
+
+## package.json support
+
+Deno also supports a `package.json` file for compatibility with Node.js
+projects. If you have a Node.js project, it is not necessary to create a
+`deno.json` file. Deno will use the `package.json` file to configure the
+project. If both a `deno.json` and `package.json` file are present in the same
+directory, Deno will import the modules in the `package.json` file and use the
+`deno.json` file for Deno-specific configurations. Read more about
+[Node compatibility in Deno](TODO:node-compat).
+
+## module imports and scopes
+
+Since version 1.30, the `deno.json` configuration file acts as an import map for
+resolving bare specifiers.
```

Comment:
I feel like this sentence can be removed as it's only relevant for old deno versions.